parser_test.go

```diff
@@ -104,3 +104,14 @@ func TestParserWithSampleRateAndTags(t *testing.T) {
 	assert.NotNil(t, valueError, "No errors when parsing")
 	assert.Contains(t, valueError.Error(), "Invalid integer", "Invalid integer error missing")
 }
+
+func TestInvalidPackets(t *testing.T) {
```

Comment:
yep, I will probably make the test more exhaustive in the future. What we really want here is some kind of fuzzer that constructs a minimally valid packet, then removes one character from it at random and expects an error. Or something along those lines, I'm not sure if the format is trivially fuzzable.